apps/editor.planx.uk/src/pages/FlowEditor/components/Settings/Team/Integrations/SubmissionEmails/index.tsx

```diff
@@ -1,71 +1,170 @@
       })}
+      getMutationVariables={(values) => {
+        const emails: SubmissionEmailMutation[] = values.input.map((email) => ({
+          submission_email: email.submissionEmail,
+          default_email: email.defaultEmail,
+          team_id: teamId,
+        }));
+
+        return { emails };
+      }}
       validationSchema={validationSchema}
-      legend="Submission information"
+      legend="Submission Emails"
       description={
         <>
           <Typography variant="body2">
-            Enter multiple email addresses to receive applications. You can
-            assign one email address per service. A default address must be
-            selected, which will be used as a fallback for all applications.
+            Add submission emails to the list. You can assign one default email
+            per service. Newly added emails will be saved when you click "Save."
           </Typography>
         </>
       }
-    >
-      {({ formik }) => (
-        <InputLabel label="Submission email" htmlFor="submissionEmail">
-          <Input
-            name="submissionEmail"
-            onChange={(e: ChangeEvent<HTMLInputElement>) =>
-              formik.setFieldValue("submissionEmail", e.target.value)
-            }
-            value={formik.values.submissionEmail}
-            errorMessage={getIn(formik.errors, "submissionEmail")}
-            id="submissionEmail"
-          />
-        </InputLabel>
+      children={({
+        formik,
+      }: {
+        formik: FormikProps<SubmissionEmailFormValues>;
+      }) => (
+        <>
+          <Typography variant="h6" style={{ marginBottom: "1rem" }}>
+            Submission Emails
+          </Typography>
+          <RadioGroup
+            value={formik.values.saved.findIndex(
+              (emailObj) => emailObj.defaultEmail,
+            )}
+            onChange={(e) => {
+              const selectedIndex = parseInt(
+                (e.target as HTMLInputElement).value,
+                10,
+              );
+              formik.setFieldValue(
+                "saved",
+                formik.values.saved.map((emailObj, index) => ({
+                  ...emailObj,
+                  defaultEmail: index === selectedIndex,
+                })),
+              );
+            }}
+          >
+            <ListManager
+              values={[...formik.values.saved, ...formik.values.input]}
+              onChange={(newValues) => {
+                const { savedEmails, newEmails } = updateEmailLists(
+                  newValues,
+                  formik.values.saved,
+                );
+
+                formik.setFieldValue("saved", savedEmails);
+                formik.setFieldValue("input", newEmails);
+              }}
+              newValue={() => ({
+                submissionEmail: "",
+                defaultEmail: false,
+                teamId: teamId,
+              })}
+              Editor={EmailsEditor}
+              maxItems={10}
```

Comment:
I think this is just a copy/paste issue - there should be no restriction on the number of values here.